plugin/trino-hive/src/main/java/io/trino/plugin/hive/acid/AcidTransaction.java

```diff
@@ -80,7 +81,7 @@ public long getWriteIdForSerialization()
     @JsonIgnore
     public boolean isAcidTransactionRunning()
     {
-        return operation == INSERT || operation == DELETE || operation == UPDATE;
+        return operation == INSERT || operation == DELETE || operation == UPDATE || operation == ANALYZE;
```

Comment:
looks like the intention here is that this place be updated whenever we add a new enum option, so we should use a switch here, without a `default` branch -- https://github.com/trinodb/trino/blob/master/DEVELOPMENT.md#avoid-default-clause-in-exhaustive-enum-based-switch-statements

Reply:
Agreed that this needs changing, but I think it might be better get this information from a new boolean constructor arg, data member and method on the `AcidOperation` enum:
`boolean AcidOperation isAcidTransaction()`
I made that change.